cloud/wfm/src/executor/ExecutorImpl.ts

```diff
@@ -0,0 +1,33 @@
+import * as Promise from 'bluebird';
+import {Process} from '../process';
+import {ProcessInstance} from '../process-instance';
+import {Task, TaskEventData, TaskStatus} from '../task';
+import {Executor} from './index';
+
+export interface InstanceRepository {
```

Comment:
Typical pattern is to have repositories in separate folder or at least separate file.